python/manylinux1/scripts/build_lz4.sh

```diff
@@ -13,6 +13,7 @@
 
 export LZ4_VERSION="1.7.5"
 export PREFIX="/usr"
+export CFLAGS="${CFLAGS} -O3 -fPIC"
```

Comment:
@xhochy ,  it seems that manylinux docker image should be updated one more time for Lz4 lib compilation with -fPIC this time, sorry about this.